_sass/_publications.scss

```diff
@@ -5,7 +5,7 @@
   
   .pill {
     display: inline-block;
-    width: 100px;
+    width: 25%;
```

Comment:
ah, sorry. I thought that first one was what it used to look like on a big screen